presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanAssert.java

```diff
@@ -49,9 +49,9 @@ public static void assertPlan(Session session, Metadata metadata, StatsProvider
         CostProvider costProvider = node -> UNKNOWN_COST;
         MatchResult matches = actual.getRoot().accept(new PlanMatchingVisitor(session, metadata, statsProvider, lookup), pattern);
         if (!matches.isMatch()) {
-            String formattedPlan = textLogicalPlan(actual.getRoot(), actual.getTypes(), metadata.getFunctionRegistry(), statsProvider, costProvider, session, 0);
+            String formattedPlan = textLogicalPlan(actual.getRoot(), actual.getTypes(), metadata.getFunctionRegistry(), actual.getStatsAndCosts(), session, 0);
```

Comment:
Once you remove the `statsAndCosts` field from the `Plan` you can simply pass `StatsAndCosts.empty()` here. Also don't forget to remove `CostProvider costProvider = node -> UNKNOWN_COST;`, as it will be not needed.

Reply:
I'll leave stats and cost with the plan, but the costProvider is already not needed.  I took it out, but somehow it got added back while I was resolving conflicts during rebase.  I'll remove :).